Add docs for swap start time (#3630)
dfinity/portal: docs/developer-docs/daos/sns/launching/launch-summary-1proposal.mdx

At this point, the SNS canisters exist and the dapp canisters are under SNS DAO control. The initial SNS neurons can use SNS proposals to take the last steps to decentralize the dapp canisters, for example by giving certain permissions to the SNS governance canister stating that these functions can only be triggered by SNS DAO proposal. One example of this is the asset canister (see details [here](../managing/sns-asset-canister.mdx)).

> ### 8. Timing Protocol for Swap Start

Reviewer: ```suggestion
### 8. Timing protocol for swap start
```

An algorithm is applied to allow at least 24 hours between the time of execution of the CreateServiceNervousSystem proposal and swap start. For example, if the start time specified in the proposal is 23:30 UTC and the proposal is adopted and executed at 23:20 UTC, then the swap start will be at 23:30 UTC the next day (i.e., in 24 hours and 10 min from the proposal execution time).

**WARNING**: The timing protocol for swap start works differently on mainnet and in testing.

On mainnet:
- Setting the start time to some value (e.g., 23:30 UTC) will allow the swap participants to be prepared for the swap in advance, e.g., by obtaining ICPs that they would like to participate with.

> - If start time is not specified, the actual start time of the swap will be chosen at random (allowing at least 24 hours and less than 48 hours, as described above).

Reviewer: ```suggestion
- If a start time is not specified, the actual start time of the swap will be chosen at random (allowing at least 24 hours and less than 48 hours, as described above).
```

In testing:
- Setting the start time to some value works the same as explained above.

> - If start time is not specified, the swap will begin immediately after the CreateServiceNervousSystem proposal is executed. This facilitates testing in an accelerated manner.

Reviewer: ```suggestion
- If a start time is not specified, the swap will begin immediately after the CreateServiceNervousSystem proposal is executed. This facilitates testing in an accelerated manner.
```

At this point, the SNS canisters exist and the dapp canisters are under SNS DAO control. The initial SNS neurons can use SNS proposals to take the last steps to decentralize the dapp canisters, for example by giving certain permissions to the SNS governance canister stating that these functions can only be triggered by SNS DAO proposal. One example of this is the asset canister (see details [here](../managing/sns-asset-canister.mdx)).

### 8. Timing protocol for swap start

> An algorithm is applied to allow at least 24 hours between the time of execution of the CreateServiceNervousSystem proposal and swap start. For example, if the start time specified in the proposal is 23:30 UTC and the proposal is adopted and executed at 23:20 UTC, then the swap start will be at 23:30 UTC the next day (i.e., in 24 hours and 10 min from the proposal execution time).

Reviewer: ```suggestion
An algorithm is applied to allow at least 24 hours between the time the CreateServiceNervousSystem proposal is executed and the swap start. For example, if the start time specified in the proposal is 23:30 UTC and the proposal is adopted and executed at 23:20 UTC, then the swap start will be at 23:30 UTC the next day (i.e., in 24 hours and 10 minutes from the proposal execution time).
```

> **WARNING**: The timing protocol for swap start works differently on mainnet and in testing.

Reviewer: ```suggestion
**WARNING**: The timing protocol for the swap start works differently on the mainnet and in testing.
```

> On mainnet:
> - Setting the start time to some value (e.g., 23:30 UTC) will allow the swap participants to be prepared for the swap in advance, e.g., by obtaining ICPs that they would like to participate with.
> - If a start time is not specified, the actual start time of the swap will be chosen at random (allowing at least 24 hours and less than 48 hours, as described above).
> 
> In testing:
> - Setting the start time to some value works the same as explained above.
> - If a start time is not specified, the swap will begin immediately after the CreateServiceNervousSystem proposal is executed. This facilitates testing in an accelerated manner.

Reviewer: ```suggestion
On the mainnet:
- Setting the start time to some value (e.g., 23:30 UTC) will allow the swap participants to be prepared for the swap in advance, e.g., by obtaining ICP tokens that they could participate with.
- If a start time is not specified, the actual start time of the swap will be chosen at random (allowing at least 24 hours and less than 48 hours, as described above).

In testing:
- Setting the start time to some value works the same as explained above.
- If a start time is not specified, the swap will begin immediately after the CreateServiceNervousSystem proposal is executed. This facilitates testing in an accelerated manner.
```
